utils/config/daos_server.yml

```diff
@@ -83,6 +83,13 @@
 #provider: ofi+verbs;ofi_rxm
 #
 #
+## Control plane network interface
+## Use a specific network interface for communications with this control plane
+## server.
```

Comment:
We should probably make it clear that this is an optional setting for use when the default behavior is unlikely to work correctly (e.g. multiple interfaces on the same subnet?).